index.js

```diff
@@ -1,6 +1,7 @@
 const http = require('http');
 const axios = require('axios');
 const mongoose = require('mongoose');
+var CronJob = require('cron').CronJob;
```

Comment:
Why you use `var` here? Please use modern approach, e.g. `const { CronJob } = require('cron');`